discord_bots/automation_bot/config.py

```diff
@@ -1,28 +1,43 @@
 import os
 import logging
+from pathlib import Path
 from pprint import pformat
 import yaml
 
+# TODO: This is ugly and needs updating if we ever move this file
+_file_depth = 3
+_file = Path(__file__).absolute()
+_top_level = _file.parents[_file_depth-1]
+
+
 if "BOT_CONFIG" in os.environ and os.path.isfile(os.environ["BOT_CONFIG"]):
-    CONFIG_PATH = os.environ["BOT_CONFIG"]
-    CONFIG_DIR = os.path.dirname(CONFIG_PATH)
+    AUTH_PATH = Path(os.environ["BOT_CONFIG"])
+    AUTH_DIR = AUTH_PATH.parent
 else:
-    CONFIG_DIR = os.path.expanduser("~/.monumenta_bot/")
-    CONFIG_PATH = os.path.join(CONFIG_DIR, "config.yml")
+    AUTH_DIR = Path("~/.monumenta_bot/").expanduser()
+    AUTH_PATH = AUTH_DIR / "config.yml"
 
-# Read the bot's config files
-with open(CONFIG_PATH, 'r') as ymlfile:
-    bot_config = yaml.load(ymlfile, Loader=yaml.FullLoader)
+# Read the bot's auth files
+with open(AUTH_PATH, 'r') as ymlfile:
+    bot_auth = yaml.load(ymlfile, Loader=yaml.FullLoader)
 
-logging.info("\nBot Configuration: %s\n", pformat(bot_config))
+logging.info("\nBot Authorization: %s\n", pformat(bot_auth))
```

Comment:
I think we explicitly do not want to print the auth config into the log. This should probably move down and print the bot config like it was doing before.

Reply:
Fair - we were printing the contents of both earlier, and I'm not sure we save the log files anywhere permanent, so I think we got away with it at the time.